Mathlib/LinearAlgebra/SesquilinearForm.lean

```diff
@@ -798,4 +798,64 @@ end CommRing
 
 end Nondegenerate
 
+namespace BilinForm
+
+lemma inner_inner_smul_sub_eq [CommRing R] [AddCommGroup M] [Module R M]
```

Comment:
The `inner` naming (used in the analysis corner of Mathlib) refers to the [Inner](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/InnerProductSpace/Basic.html#Inner) typeclass so we shouldn't use it here.
```suggestion
lemma apply_smul_sub_smul_sub_eq [CommRing R] [AddCommGroup M] [Module R M]
```